### ISSUE TITLE
[Suggesstion] Add icon themes or use system icons

### ISSUE THREAD

**Issue by sixsixfive** [sk1project/sk1-wx#137]
For me one of the biggest advantages of lInux compared to windows is that you can make all applications to use the same icons and style. however Sk1 uses own icons which makes it really hard for me to use it with other graphic applications.

![abc](https://user-images.githubusercontent.com/40307321/41492239-6cd9f6ea-70c3-11e8-8af1-7a3c3a10b4e7.png)

**Comment by unixino** [comment]
Hi 
Take a look at #107

**Comment by sixsixfive** [comment]
Sorry but that the complete opposite what I suggest.

I would like to use the same theme(thts already the case since it uses my GTK2 theme) and icons as my system uses. In that way all my applications will use the same icons for the same tool(as it is already the case for GIMP and Inkscape.)

there are two ways to do that, either by supporting extra application themes like GIMP>=2.10, Scribus or Libreoffice do or by loading the icons from the X11/freedesktop icon theme(like inkscape does if an icon with that GTK_STOCK_ID exists in the theme: http://wiki.inkscape.org/wiki/index.php/Themable_icons )

**Comment by Imerion** [comment]
I agree with this, using the system icon theme is very helpful and also results in a uniform look of applications no matter what style you like.
